wpilibc/src/main/native/cpp/controller/PIDController.cpp

```diff
@@ -18,6 +19,10 @@ using namespace frc2;
 PIDController::PIDController(double Kp, double Ki, double Kd,
                              units::second_t period)
     : m_Kp(Kp), m_Ki(Ki), m_Kd(Kd), m_period(period) {
+  if (period == 0_s) {
```

Comment:
Because the robot code doesn't crash here like Java would, we should set the period to the default of 20_ms and potentially let the user know about that.